pynsee/localdata/get_new_city.py

```diff
@@ -7,10 +7,15 @@
 
 from pynsee.utils._request_insee import _request_insee
 
+import logging
+logger = logging.getLogger(__name__)
 
 @lru_cache(maxsize=None)
 def _warning_get_new_city():
-    print("\ndate is None, by default it supposed to be ten years before current year")
+    logger.info(

```

Comment:
should be a warning

Reply:
This might need a refacto altogether for consistency. @hadrilec I think you started to change that behaviour in multiple functions to be in accordance with INSEE's API (ie current date) ?

Reply:
@hadrilec I thought we had answered all question, but I'm actually not sure we dealt with this one...

Reply:
@hadrilec up: should this be a warning or is info fine?